Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,29 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const result =
+    // step 1.2.4
+    Math.floor(
```

Comment:
Rounding this number to two decimals is only required for presentation purpose (i.e. showing the result to the user). By rounding at this stage the result this function can no longer be used for accurate subsequent calculations (because of loss of precision). It is best to delay the rounding until the result must be presented (i.e. outside of this function).